resources/sts/4.11/sts_installer_privatelink_permission_policy.json

```diff
@@ -0,0 +1,17 @@
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+		    "ec2:ModifyVpcEndpointServiceConfiguration",
+		    "route53:ListHostedZonesByVPC",
+		    "route53:CreateVPCAssociationAuthorization",
+		    "route53:AssociateVPCWithHostedZone",
+		    "route53:DeleteVPCAssociationAuthorization",
+		    "route53:DisassociateVPCFromHostedZone"
```

Comment:
Should we add all these new permissions to the sts_installer_permission_policy.json as well?
To have the installer policy equivalent to the three new policies?